vignettes/python_dependencies.Rmd

```diff
@@ -152,18 +153,22 @@ Declared Python package dependencies should have the following format:
 
 - **package**: The name of the Python package.
 
+- **version**: The version of the package that should be installed.
```

Comment:
We should emphasize that this should only be used in cases where it's known that more recent versions have breakage. As such, conflicts here will be resolved in favor of the older version. People should just understand that in order to ensure the broadest compatibility this should work much like CRAN (where you always get the latest version and it's assumed they work / work together) and this is an exceptional escape hatch for when that isn't the case.